bonus.mobile-app/CrmApp/src/helpers/authService.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { USER_KEY } from '../config';
+import { canInstrument } from 'babel-jest';
+
+class AuthService {
+
+    constructor(){
+        this.basicUrl = 'http://localhost:8005';
+    }
+
```

Comment:
you can make the functions in this class static, 
since you don't really have the "this" concept. 
this way, you won't have to call the constructor on each file you use it